src/App.tsx

```diff
@@ -1,34 +1,40 @@
-import { useState } from 'react';
-
-import { Content } from '@/components/Content';
-import { Sidebar } from '@/components/Sidebar';
-import { Credentials } from '@/pages/Credentials';
 import './App.scss';
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
 
-function App() {
-  const [activeContent, setActiveContent] = useState('My Personal Project-Credentials');
-
-  const renderContent = () => {
-    // Temporary function to render content based on activeContent.
-    // This will change and be moved to a card based approach for an active workspace.
-    const [workspaceProject, workspaceArea] = activeContent.split('-');
-
-    switch (workspaceArea) {
-      case 'Credentials':
-        return <Credentials workspaceProject={workspaceProject} />;
-      case 'Workspaces':
-        return <div>Workspaces</div>;
-      default:
-        return <div>Default</div>;
-    }
-  };
+import TopBar from './components/TopBar/TopBar';
+import 'react-tabs/style/react-tabs.css';
```

Comment:
More an FYI, I tend to have CSS/SCSS files all imported after any JavaScript imports. I also think it is good to have them co-located, whereas the App style as at the top, then the tabs is in the middle. Just makes it more predictable and obvious, my opinion only.

I also notice we're not using **Absolute** imports, my own personal view is to use absolute imports for all but sibling components, then if we ever re-structure the app and then it just works.

Reply:
Updated the imports